src/main/java/mate/project/controller/BookController.java

```diff
@@ -19,6 +22,7 @@
 import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.server.ResponseStatusException;
 
+@Tag(name = "Book Management", description = "Endpoints for managing books")
```

Comment:
Consider providing more detailed descriptions in your Swagger annotations. 'Endpoints for managing books' is a start, but try to include more specific details about what each endpoint does.